components/Datatable.tsx

```diff
@@ -99,10 +99,10 @@ const DataTable: React.FC<DataTableProps> = ({ data, columns, loading }) => {
         </Thead>
         <Tbody {...getTableBodyProps()}>
           {!loading &&
-            rows.map(row => {
+            rows.map((row, i) => {
               prepareRow(row)
               return (
-                <Tr {...row.getRowProps()} key={row.id}>
+                <Tr {...row.getRowProps()} key={i}>
```

Comment:
Are those fixing the nth-child errors that keep popping up?

Reply:
No, this will fix the error when the keys are not unique

the `nth-child` error has to do with css on line 33 of this same file.